model/client4.go

```diff
@@ -543,6 +543,12 @@ func (c *Client4) CreateUserWithHash(user *User, hash, data string) (*User, *Res
 	}
 }
 
+// CreateUserWithHash creates a user in the system based on the provided user struct and hash created.
```

Comment:
It won't actually work with the old hash. You need to supply a token. So not sure how valuable this backwards compatibility is. Should at least update the comment to warn this.